### ISSUE TITLE
Lasso behavior partially broken in 3.1.12

### ISSUE THREAD

**Issue by GTR8000** [cpriest/SnapLinksPlus#495]
With 3.1.11, I could start at the top of a full page of links, right-click to begin lassoing those links, and as I went down the list the page would scroll along with the lasso, allowing me to select everything on that web page.

3.1.12 breaks that; it now stops at the bottom of the **visible content** on the page and will not allow the page to scroll to the bottom while continuing to select links within the lasso.

Rolled back to 3.1.11 and confirmed that the original/preferred scrolling behavior was restored.

Tested in both Firefox 130.0.1 and Waterfox G6.0.19

**Comment by cpriest** [comment]
As a workaround you should be able to scroll with the mouse wheel or other methods of scrolling down during a drag.

**Comment by Sand-Mage** [comment]
As an actual workaround I just installed the previous version of snaplinks and disabled updates.

**Comment by GTR8000** [comment]
Correct, the most effective workaround was to simply roll back to 3.1.11 and disable auto-update until the issue is fixed in 3.1.13

**Comment by hagenfenris** [comment]
Hi,
just updated to 3.1.13 an the scrolling-problem still exists.
selecting multiple links with right-click and scrolling down dosn`t work.
Starting from the end of the website and scrolling up, works!

**Comment by GTR8000** [comment]
Yep, still broken in 3.1.13, tested in both Waterfox 6.5.0 (based on Firefox 128 ESR) as well as Firefox 132.0.1 😖
